connect/mirror/src/main/java/org/apache/kafka/connect/mirror/MirrorCheckpointConfig.java

```diff
@@ -166,6 +170,25 @@ Duration consumerPollTimeout() {
         return Duration.ofMillis(getLong(CONSUMER_POLL_TIMEOUT_MILLIS));
     }
 
+    public boolean validate() {
+        Boolean emitCheckpointsValue = this.getBoolean(EMIT_CHECKPOINTS_ENABLED);
+        Boolean syncGroupOffsetsValue = this.getBoolean(SYNC_GROUP_OFFSETS_ENABLED);
+
+        if (!emitCheckpointsValue && !syncGroupOffsetsValue) {
+            LOG.warn("MirrorCheckpointConnector can't run without both " + SYNC_GROUP_OFFSETS_ENABLED + ", " +
+                    EMIT_CHECKPOINTS_ENABLED + " set to false");
+            return false;
+        }
+
+        boolean requireOffsetSyncs = emitCheckpointsValue || syncGroupOffsetsValue;
```

Comment:
As `!emitCheckpointsValue && !syncGroupOffsetsValue` return before, `requireOffsetSyncs` is always true, right?

Reply:
With above comment, I feel we don't need local variables `emitCheckpointsValue` and `syncGroupOffsetsValue`

Reply:
refactored this part now